cloudnetpy/instruments/disdrometer.py

```diff
@@ -443,10 +445,18 @@ def _parse_int(row: np.ndarray) -> np.ndarray:
         units="m",
         comment="Upper and lower bounds of diameter interval.",
     ),
+    "rainfall_rate_1min_total": MetaData(
+        long_name="Precipitation rate",
+        units="m s-1",
+    ),
     "rainfall_rate": MetaData(
         long_name="Precipitation rate",
         units="m s-1",
     ),
+    "rainfall_rate_1min_solid": MetaData(
+        long_name="Precipitation rate",
+        units="m s-1",
+    ),
```

Comment:
Should these have different `long_name`?

Reply:
Oh yes, sorry, my fault.